parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -214,15 +214,15 @@ cleanUpImages() {
     export targetImage
     function cleanupImagesRun() {
         {{if NeedsContainerd}}
-        images_to_delete=$(ctr --namespace k8s.io images list | grep -vE "${KUBERNETES_VERSION}$|${KUBERNETES_VERSION}.[0-9]+$|${KUBERNETES_VERSION}-|${KUBERNETES_VERSION}_" | grep ${targetImage} | awk '{print $1}')
```

Comment:
why do we need to remove the trailing $

Reply:
reverted - bad copy/paste from a wip script I was trying out on an actual aks base VM